### DISCUSSION TITLE
How to set up the base URL (the ORIGIN configuration) in the env_variables.yaml file?

### DISCUSSION THREAD

**Discussion by JiQiRenAlpha** [FIRST-Tech-Challenge/fmltc#264] [Q&A]
Hi everyone, 
I ran into some issues when following the instructions to set up a Google Cloud Project service URL.
I wonder if you can help me to understand how to set up the base URL which is needed in the env_variables.yaml file.
This is the instructions in the env_variables.yaml file. However, there is no URL on the mentioned page in the top right corner.
![image](https://user-images.githubusercontent.com/111622875/187017463-8edc3061-1864-40ab-ae1a-ee5036b2a6a5.png)
In fact, the mentioned web page will start to show an URL only after we fill in a URL for the ORIGIN property first, not the other way around.
Is there any way to manually configure this base URL?
I tried to follow the prompts in this page to construct the base URL.
https://cloud.google.com/appengine/docs/legacy/standard/python/how-requests-are-routed#:~:text=Once%20your%20app%20is%20running%20in%20App%20Engine%2C,version%20that%20you%20have%20configured%20to%20receive%20traffic.
The above page mentions the routing URL has below format, is this correct?                                                                   
                 https://PROJECT_ID.REGION_ID.r.appspot.com
Thank you very much for your help!

**Comment by lizlooney** [comment]
Yes, the url format is https://project_id.region_id.r.appspot.com/
